transcendence-app/src/authorization/guards/base-policies.guard.ts

```diff
@@ -39,7 +42,14 @@ export abstract class PoliciesGuard implements CanActivate {
         CHECK_POLICIES_KEY,
         ctx.getHandler(),
       ) || [];
-    const ability = await this.getAbilityFromContext(ctx);
+    const request: Request =
+      ctx.getType() === 'http'
+        ? ctx.switchToHttp().getRequest()
+        : ctx.switchToWs().getClient().request;
+    const param =
+      this.reflector.get<string>(CONFIG_PARAM_KEY, ctx.getHandler()) ||
+      'chatroomId';
```

Comment:
Because in the only place where I'm actually using this, namely : `transcendence-app/src/chat/chat.controller.ts` L 223, `chatroomId` is the parameter it needs to read. I'm also passing it, so it should never hit the `|| 'chatroomId'` part, but type consistency asks that I give it a default. If we ever want to use it in other places, we can revisit my choice.